src/react-components/room/UserProfileSidebarContainer.js

```diff
@@ -146,5 +151,12 @@ UserProfileSidebarContainer.propTypes = {
   onBack: PropTypes.func,
   onClose: PropTypes.func,
   onCloseDialog: PropTypes.func.isRequired,
-  showNonHistoriedDialog: PropTypes.func
+  showNonHistoriedDialog: PropTypes.func,
+  minVolume: PropTypes.number,
```

Comment:
Confused about these properties. I was at first surprised why they would exist, but then it looks like they arent being used so I am guessing they just got left in by mistake

Reply:
Yep, I think I left that from an initial implementation. Removed.